src/main/java/com/avail/utility/Casts.java

```diff
@@ -52,7 +52,7 @@ private Casts () {}
 	 * @param <O> The output type.
 	 */
 	@SuppressWarnings("unchecked")
-	public static <I, O> O cast (final I value)
+	public static <I, O extends I> O cast (final I value)
```

Comment:
Statically enforces narrowing conversions only — nice tightening.

Reply:
Unfortunately, Java's type deduction machinery is so weak that it can't figure out a lot of obvious narrowing cases.  Still, this change adds some nice extra safety.